src/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -377,12 +377,25 @@ public static void ToString(Guid guid, string format, string expected)
             Assert.Equal(expected, formattable.ToString(format, null));
         }
 
+        public static IEnumerable<object[]> InvalidFormat_TestData()
+        {
+            yield return new object[] { "a" };
+            yield return new object[] { "c" };
+            yield return new object[] { "e" };
+            yield return new object[] { "m" };
+            yield return new object[] { "o" };
+            yield return new object[] { "q" };
+            yield return new object[] { "w" };
```

Comment:
Is there something special about all of these additional values?  Why are they beneficial?

Reply:
Because a switch statement generates a check of `if (c < Value1)...` and then `if (c < Value2)...` etc. this means that we get 100% branch coverage on each potential value of the format ('b', 'd', 'n', 'p', 'x')

Reply:
Hmm.  Don't love it, but ok.  Thanks.